Bio/PDB/ccealignmodule.c

```diff
@@ -425,6 +568,21 @@ findPath(
         } // ROF -- end for iB
     }     // ROF -- end for iA
 
+    double zScoreBuffer[MAX_PATHS];
+
+    for (int i = 0; i < bufferSize; i++) {
```

Comment:
The Z-score is calculated after the fact and not taken into account during the path finding process right? Just to confirm that this is the case, otherwise we might want to push this up into the main loop.

Reply:
That is correct.

"Extension of the alignment path is based solely on the distance criteria. Neither the size of the gap, nor the statistical significance of the alignment path is considered at this point in the analysis."